Formula/l/logalize.rb

```diff
@@ -0,0 +1,39 @@
+class Logalize < Formula
+  desc "Fast and extensible log colorizer. Alternative to ccze"
+  homepage "https://github.com/deponian/logalize"
+  url "https://github.com/deponian/logalize.git",
+    tag:      "v0.4.1",
+    revision: "2d74f36f36037d1a09371f044a85b2b8f192905f"
+  license "MIT"
+  head "https://github.com/deponian/logalize.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "VERSION=#{version}", "build"
+    bin.install "dist/#{version}/logalize"
+
+    system "make", "VERSION=#{version}", "manpage"
+    man1.install "manpages/logalize.1.gz"
+
+    system "make", "completions"
+    bash_completion.install "completions/logalize.bash" => "logalize"
+    zsh_completion.install "completions/logalize.zsh" => "_logalize"
+    fish_completion.install "completions/logalize.fish"
```

Comment:
```suggestion
    ldflags = "xxx"
    system "go", "build", *std_go_args(ldflags:), "./cmd"

    generate_completions_from_executable(bin/"logalize", "completion")
```

Reply:
I's much simpler for me as developer of the application to build everything using `make`. This way I'll be able to change building process in the future without changing the formula